charts/zac/ci/helm-testing-values.yaml

```diff
@@ -104,6 +104,9 @@ smartDocuments:
   # -- Fixed username for authentication
   fixedUserName: "fake"
 
+# -- API key for authentication of internal ZAC endpoints
```

Comment:
is this aligned properly? or is it the view that strips leading blanks ?

Reply:
or is it a top-level item, then is it in the right place?

Reply:
You mean the `--`. Well, I noticed that none of these single line property descriptions end up in the Helm README so I thought, maybe this notation fixes that. No idea really. And it worked! Now the description does end up in the README. Do you know more about this @niorg ? Is that how this Helm chart readme GitHub action works? If so, we should do the same for all Helm chart values.

Reply:
It would seem so indeed: https://github.com/norwoodj/helm-docs

Via: https://github.com/losisin/helm-docs-github-action

Reply:
This is just the Helm test file btw. The real fix (with the `# --`) is in the values.yaml file.

Reply:
So, the `# --` format is for field descriptions instead of normal comments and we should use that everywhere. For a different PR. :-)